modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -42,6 +42,8 @@ GrainTracker::GrainTracker(const InputParameters & parameters) :
 {
   if (!_is_elemental && _compute_op_maps)
     mooseError("\"compute_op_maps\" is only supported with \"flood_entity_type = ELEMENTAL\"");
+
+  _empty_2.resize(_n_vars, libMesh::invalid_uint);
```

Comment:
Well, the default does not really help. I have to explicitly check if the grain_num is valid (something other than `libMesh::invalid_uint`) in order to avoid seg-fault.